### ISSUE TITLE
Comparison table of installation methods is not accessible

### ISSUE THREAD

**Issue by steverep** [home-assistant/home-assistant.github.io#35323]
### Feedback

The SVGs used to fill out the comparison table of installation methods are inaccessible to assistive technology like screen readers.  They should all have a label like "yes" or "no".

### URL

https://www.home-assistant.io/installation/

### Version

2024.10.3
